torchvision/models/detection/faster_rcnn.py

```diff
@@ -402,7 +402,7 @@ def fasterrcnn_resnet50_fpn(
 
 
 def _fasterrcnn_mobilenet_v3_large_fpn(
-    weights_name,
+    arch,
```

Comment:
Weights name or key might be more accurate in my opinion. The reason for that is that this key represents a architecture plus dataset

Reply:
I agree with your comment for this specific case. The problem is that arch is actually used in many places as:
https://github.com/pytorch/vision/blob/f15ba56fec946e97a1f675e16e0c640a6284d337/torchvision/models/mobilenetv3.py#L312-L314

Here it does mean architecture. What I'm doing in this PR, is just aligning the name of the variable everywhere for the shake of consistency.

I can undo the changes here but I propose to not sweat about it because all the `weight_names` will be gone anyway with the Multi-weight API in the next couple of weeks. Thoughts?